resources/js/pages/Characters.vue

```diff
@@ -3,9 +3,9 @@
         <div id="characters" class="relative bg-dark-gray2-75 p-4 rounded-lg m-auto mt-4 max-w-party min-h-screen">
 
             <tabs class="hidden sm:block"
-                  :tabs="[sheet.game === Game.fh ? $t('Campaign sheet') : $t('Party sheet'), $t('Characters'), $t('Items')]"
```

Comment:
This tabs component is getting a bit messy, I think we need a prop with an array of tab class instances, each with a name, icon and url.